Dockerfile

```diff
@@ -1,15 +1,27 @@
-FROM --platform=$TARGETPLATFORM tapyrus/builder:v0.2.0 as builder
+FROM --platform=$TARGETPLATFORM tapyrus/builder:v0.3.0 as builder
 ARG TARGETARCH
 
 ENV LC_ALL C.UTF-8
-ENV TAPYRUS_CONFIG "--disable-tests --disable-bench --disable-dependency-tracking  --bindir=/tapyrus-core/dist/bin  --libdir=/tapyrus-core/dist/lib --enable-zmq --enable-reduce-exports --with-incompatible-bdb --with-gui=no CPPFLAGS=-DDEBUG_LOCKORDER"
+ENV TAPYRUS_CONFIG "--disable-tests --disable-bench --disable-dependency-tracking  --bindir=/tapyrus-core/dist/bin  --libdir=/tapyrus-core/dist/lib --enable-zmq --enable-reduce-exports --with-incompatible-bdb --with-gui=no --enable-cxx --disable-shared --disable-replication --with-pic CPPFLAGS=-DDEBUG_LOCKORDER"
+
+ENV BUILD_PACKAGES "build-essential libtool autotools-dev automake pkg-config bsdmainutils curl git ca-certificates ccache g++ make cmake gdb gdbserver rsync zip openssh-server lldb ssh python3 bison"
+
+ENV TRACE_PACKAGES "bpfcc-tools systemtap-sdt-dev python3-bpfcc"
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
+RUN apt-get install --no-install-recommends --no-upgrade -qq $PACKAGES
+RUN apt-get install --no-install-recommends --no-upgrade -qq $BUILD_PACKAGES
+RUN apt-get install --no-install-recommends --no-upgrade -qq $TRACE_PACKAGES
+RUN apt-get install --no-install-recommends --no-upgrade -qq linux-headers-generic
```

Comment:
The installation of these packages was done by tapyrus-builder, so it is not necessary or they should be installed with tapyrus-builder.